src/plugins/cosmos/CosmosAssetAccountDetails.tsx

```diff
@@ -25,7 +25,10 @@ export const CosmosAssetAccountDetails = ({ assetId, accountId }: AssetDetailsPr
         <AssetChart accountId={accountId} assetId={assetId} isLoaded={true} />
         {accountId && <AccountAssets assetId={assetId} accountId={accountId} />}
         <AssetAccounts assetId={assetId} accountId={accountId} />
-        <StakingOpportunities assetId={assetId} />
+        {
+          /* Staking is only supported for cosmos (not osmosis yet) */ assetId ===
+            'cosmos:cosmoshub-4/slip44:118' && <StakingOpportunities assetId={assetId} />
```

Comment:
Check this from inside the StakingOpportunities usingfeature flag and return null if not on for osmo